commands/view-state.js

```diff
@@ -36,8 +35,21 @@ async function viewState(options) {
     const { accountId, prefix, finality, blockId, utf8 } = options;
     const near = await connect(options);
     const account = await near.account(accountId);
-
-    let state = await account.viewState(prefix, { blockId, finality });
+    if (finality && blockId) {
+        console.error('Only one of --finality and --blockId can be provided');
+        process.exit(1);
+    } else if (!finality && !blockId) {
+        console.error('Must provide either --finality or --blockId');
+        process.exit(1);
+    }
+    // near-api-js takes block_id instead of blockId
+    let block_id = blockId;
```

Comment:
YArgs automatically creates properties on options for both the hypenated and non-hypenated strings for each option -- there's no need to have a local block_id variable here at all, you can just change L35 to reference options.block-id :)

Reply:
Ah, my apologies -- I just realised that this was a snake_case conversion, not a dashed-property :+1: nevermind this one :)